main.go

```diff
@@ -28,7 +35,38 @@ func main() {
 		cancel()
 	}()
 
-	go http.ListenAndServe(":80", api.New())
+	// Load Environment Variables
```

Comment:
Generally, comments should add value by telling us why code does what it does, not what does the code do. We already know what the code does because we can read the code itself. There are exceptions if the code does something particularly tricky, but I don't think this is one of those cases. 

The func name `FromEnv` and the context around it, like the log message, already tells the reader that these lines of code load a config from env.